src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -772,7 +774,7 @@ class CreateVmModal extends React.Component {
         }
         super(props);
         this.state = {
-            inProgress: false,
+            inProgress: 0,
```

Comment:
Please define and use a named constant for "0" as well; like "INVALID" or "DEFINE"; and `inProgress` still sounds boolean-y, maybe rename to "stage" or "step"?